From: Jason J. Herne <jjherne@linux.vnet.ibm.com>
Subject: s390: kvm: Cpu model support for msa6, msa7 and msa8
Patch-mainline: No, feature masks are handled differently now
References: FATE#324069, LTC#159031

Summary:     kvm-kernel: VS1606 zMidas cpu model checking backport
Description: Provide cpu model support for msa 6/7/8

Upstream-Description:

             s390: kvm: Cpu model support for msa6, msa7 and msa8

             msa6 and msa7 require no changes.
             msa8 adds kma instruction and feature area.

             Signed-off-by: Jason J. Herne <jjherne@linux.vnet.ibm.com>
             Reviewed-by: Christian Borntraeger <borntraeger@de.ibm.com>
             Signed-off-by: Christian Borntraeger <borntraeger@de.ibm.com>


Signed-off-by: Jason J. Herne <jjherne@linux.vnet.ibm.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 arch/s390/kvm/kvm-s390.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/s390/kvm/kvm-s390.c
+++ b/arch/s390/kvm/kvm-s390.c
@@ -122,7 +122,7 @@ struct kvm_stats_debugfs_item debugfs_en
 unsigned long kvm_s390_fac_list_mask[] = {
 	0xffe6000000000000UL,
 	0x005e000000000000UL,
-	0x3000000000000000UL,
+	0x3000200000000000UL,
 };
 
 unsigned long kvm_s390_fac_list_mask_size(void)
